.github/workflows/ci-test-hosted.yml

```
@@ -182,8 +182,14 @@ jobs:
             echo "COMMIT_INFO_MESSAGE=Manual workflow run for Hosted tests" >> $GITHUB_ENV
           fi
 
+      - name: Monitor Resource Usage
+        run: |
+          echo "Monitoring system metrics during Cypress tests..."
+          nohup bash -c 'while true; do echo "==== CPU & RAM Usage ===="; top -b -n 1 | head -n 10; echo "==== Disk Usage ===="; df -h; sleep 10; done' > resource_usage.log &
+          echo $! > resource_monitor_pid    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)